src/pages/ListScholarships.jsx

```diff
@@ -9,34 +9,50 @@ import {
   Toolbar,
   useMediaQuery,
   useScrollTrigger,
+  Chip,
 } from '@mui/material';
 import FilterBar from '../components/FilterBar';
 import FilterPanel from '../components/FilterPanel';
 import ScholarshipList from '../components/ScholarshipList';
 import useQueryParams from '../lib/useQueryParams';
 import { DEADLINE_ASC, getDir, getField } from '../lib/sortOptions';
 import { HeaderSkeleton } from '../components/Header';
+import GradeLevel from '../types/GradeLevel';
 
 const drawerWidth = 360;
 
+function FilterChip({ label, deleteFn }) {
+  return (
+    <Chip
+      key={label}
+      label={label}
+      color="primary"
+      onClick={deleteFn}
+      onDelete={deleteFn}
+      sx={{ mr: 2, mt: 3 }}
```

Comment:
This seems like something a [Stack](https://mui.com/components/stack/#main-content) might be able to help with. 🤔 

Also, since this is a two property thing for items that otherwise go right next to each other, why not do something like this:
```jsx
const filterChips = {};
if (Number.isInteger(minAmount)) { filterChips[`Min ${minAmount}`]: /* delete function */ ;}
// same thing for maxAmount

majors?.forEach((m) => filterChips[m] = /* delete function */; );
// same thing for grades.

return (
  // ...
  {filterChips.items().map(([label, deleteFn]) => 
    <Chip />
  }
  // ...
);
```

This'll also make it a little easier to add future filters because JSX doesn't need to be touched.